Source/WTF/wtf/MemoryPressureHandler.cpp

```diff
@@ -145,11 +145,13 @@ MemoryPressureHandler::MemoryPressureHandler()
 
 void MemoryPressureHandler::setShouldUsePeriodicMemoryMonitor(bool use)
 {
+#if !PLATFORM(WPE) || !ENABLE(MALLOC_HEAP_BRAKDOWN)
```

Comment:
We're just building wpe with this repository. You don't need the PLATFORM(WPE) guard.

Reply:
Also there's a typo in how you wrote BREAKDOWN.

I understand that enabling MALLOC_HEAP_BREAKDOWN is causing isFastMallocEnabled() to return false and that's why you need this?

Reply:
oh, sorry for typo, I fixed it locally but didn't include in commit. I will fix both.

Right, with MALLOC_HEAP_BREAKDOWN enabled, isFastMallocEnabled() returns false. It's calculated inside:
```
bool Environment::computeIsDebugHeapEnabled()
{
    if (isMallocEnvironmentVariableImplyingSystemMallocSet())
        return true;
    ...

        return true;
#endif

#if BENABLE_MALLOC_HEAP_BREAKDOWN
    return true;
#endif

    return false;
}
```